### ISSUE TITLE
How to consume render callbacks with multiple arguments?

### ISSUE THREAD

**Issue by williamboman** [Astrocoders/epitath#20]
From reading the code, it looks like `epitath` only passes the first render prop argument to the generator - how can a render prop component that calls the render function with multiple arguments be consumed? Is this an oversight <sup>sorry for wording</sup> or is it by design?

**Comment by grsabreu** [comment]
It's by design! We recommend people creating their own wrappers https://github.com/Astrocoders/epitath/issues/1#issuecomment-418912767 to avoid bloating to much the API

```js
const MutationWrapper = ({children, ...props}) =>
  <Mutation {...props}>{(mutate, result) => props.children({ mutate, result })}<Mutation/>

...
const { mutate, result } = yield <MutationWrapper any props />
```

**Comment by williamboman** [comment]
Cool! That's exactly how I solved it. Feel free to close this issue, I'll leave it open

**Comment by guilhermedecampo** [comment]
@williamboman how about adding this information in the README? Can you create a PR? 🚀

**Comment by grsabreu** [comment]
Closed by #21
